src/spinner/spinner.ts

```diff
@@ -4,6 +4,8 @@ import { classMap } from 'lit/directives/class-map.js';
 
 export class Spinner extends LitElement {
   @property() variant: 'normal' | 'inverse' | 'inherit' = 'inherit';
+  @property() size: 'small' | 'medium' | 'large' = 'medium';
```

Comment:
I think you might need to use `reflect` here because you style the spinner based on the size attribute. so you need to reflect the property to the attribute. 

it should be applied to `variant` too.

Reply:
Fixed